README.md

```diff
@@ -32,9 +32,11 @@ repository. We'll review the pull request and get back to you within a few
 days.
 
 To join the ecosystem, your project must:
-  - Have an open-source license
-  - Adhere to our [code of conduct](./CODE_OF_CONDUCT.md)
-  - Build on, interface with, or extend Qiskit in some way
+  - Build on, interface with, or extend the Qiskit SDK in some way.
+  - Be compatible with Qiskit 1.0 (or newer).
+  - Have an [OSI-approved](https://opensource.org/license?categories=popular-strong-community) open-source license (preferably Apache 2.0 or MIT).
+  - Adhere to our [code of conduct](./CODE_OF_CONDUCT.md).
```

Comment:
Should we offer that you can have an additional code of conduct?

Reply:
You mean they can enforce their own in addition to ours?

Reply:
Exactly. Like theirs can be a superset.